doc/conf.py

```diff
@@ -791,6 +791,15 @@ def reset_warnings(gallery_conf, fname):
         'ignore', message="can't resolve package from", category=ImportWarning)
     warnings.filterwarnings(
         'ignore', message='.*mne-realtime.*', category=DeprecationWarning)
+    warnings.filterwarnings(
```

Comment:
This was errantly duplicated

Also perhaps you don't need these changes anymore? The latest CircleCI PR doc builds have been green at least

Reply:
I reverted the changes and now we get the deprecation warnings again and an exception :(

Reply:
looks like you reverted too much!  the jinja glue line was supposed to stay...  it's just the accidentally dupllicated mne-realtime line that needed reverting

Reply:
I thought even the jinja-glue wouldn't be needed since builds are passing on other PRs. But maybe this PR changes something that makes it necessary? Or the caching is making different builds use different versions of Sphinx or something?

Reply:
version of sphinx is the same between this failed circleCI build and the successful one on #9316 (both use `4.0.0b1`).  Jinja is different (`3.0.0rc1` here versus `3.0.0a1` there)

Reply:
the jinja error is happening on #9292 now also.

Reply:
Feel free to port the fix over. Personally I would do it just by adding another line to the `DeprecationWarnings` list a few lines above

Reply:
(around line 770 of conf.py)